RocketChat/Rocket.Chat.ReactNative
# [IMPROVE] Use UI Elements from react-navigation

## Proposed changes
Use UI Elements from react-navigation

## Issue(s)	


## How to test or reproduce

### Test if header is working fine for both normal and search:
- Discussions View (room -> actions -> discussions)
- Thread View ( room -> thread button )
- Team view (team -> teams button )
- Canned Responses view (on a omnichannel chats in progress -> actions -> canned responses)

### Others
- Share Extension:
	- Check if the functionality still works fine;
	- Check if the image/video is centralized correctly 
- Attachment View:
	- Test the name on header (big names)
	- Test if the image is centralized right 
- Rooms list
	- search -> check if the search is working on the android, without hiding the text to the left
	- change the themes, and check if the headers is ok with each one
	- check a server with long name
	- check if the dropdown is working fine
- RoomView
	- Check long names 
	- Discussions, Threads and other room types should behave as expected
	- 

## Screenshots

## Types of changes




## Checklist



## Further comments
